CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Smoke tests of the Bernoulli multi-armed bandit example have meaningful MC run and
```

Comment:
wouldnt mention this very minor change in the CHANGELOG, if the check in the pipeline fails its fine, this is just an optional reminder

Reply:
I reverted the last commit updating the CHANGELOG.md.